amundsen_application/models/user.py

```diff
@@ -13,76 +13,68 @@
 
 
 class User:
-    # TODO: alphabetize after we have the real params
     def __init__(self,
+                 email: str = None,
+                 employee_type: str = None,
                  first_name: str = None,
+                 full_name: str = None,
+                 github_username: str = None,
+                 is_active: bool = True,
                  last_name: str = None,
-                 email: str = None,
-                 display_name: str = None,
+                 manager_fullname: str = None,
                  profile_url: str = None,
-                 user_id: str = None,
-                 github_name: str = None,
-                 is_active: bool = True,
-                 manager_name: str = None,
                  role_name: str = None,
-                 slack_url: str = None,
-                 team_name: str = None) -> None:
+                 slack_id: str = None,
+                 team_name: str = None,
+                 user_id: str = None) -> None:
+        self.email = email
+        self.employee_type = employee_type
         self.first_name = first_name
+        self.full_name = full_name
+        self.github_username = github_username
+        self.is_active = is_active
         self.last_name = last_name
-        self.email = email
-        self.display_name = display_name
+        self.manager_fullname = manager_fullname
         self.profile_url = profile_url
-
-        # TODO: modify the following names as needed after backend support is implemented
-        self.user_id = user_id
-        self.github_name = github_name
-        self.is_active = is_active
-        self.manager_name = manager_name
         self.role_name = role_name
-        self.slack_url = slack_url
+        self.slack_id = slack_id
         self.team_name = team_name
-        # TODO: frequent_used, bookmarked, & owned resources
+        self.user_id = user_id
+        # TODO: Add frequent_used, bookmarked, & owned resources
 
     def to_json(self) -> Response:
         user_info = dump_user(self)
         return jsonify(user_info)
 
 
 class UserSchema(Schema):
-    first_name = fields.Str(allow_none=True)
-    last_name = fields.Str(allow_none=True)
-    email = fields.Str(allow_none=True)
-    display_name = fields.Str(required=True)
-    profile_url = fields.Str(allow_none=True)
-
+    email = fields.Str()
```

Comment:
Why removing `allow_none=True`?

Reply:
Removing `allow_none=True` didn't seem to change anything. I'm actually not sure what these `fields.Str()` do or if we even need them. It seems like the `UserSchema` class is just used to preprocess/validate a `Dict` before passing it to the `User` class.